_includes/swagger-method.html

```diff
@@ -0,0 +1,76 @@
+
+{% assign route   = include.r %}
+{% assign swagger = include.s %}
+
+<div class="swagger-paths">
+    {% for method in route[1] %}
+    <div class="swagger-method swagger-method-{{ method[0] }}">
+        <h3 class="swagger-method-title">
```

Comment:
I'd use `h4` here. You can style `.swagger-tag` to become bigger if you want. It would be nice to have an `id` attribute to link it via an anchor.

Reply:
Done